update pokemon in index.html

donppyl opened the pull request:
Added Pokemon in index.html

AM1CODES commented:
Can you brief me about the pokemon that you have added and the changes that you have made

donppyl commented:
> Can you brief me about the pokemon that you have added and the changes that you have made

Sure, I added Pokemon Horsea to the list and added a photo and description

AM1CODES commented:
Okay! I am not sure if horsea is present in the website, so i will just merge it for now. Thank you for contributing
